resolver/Cargo.toml

```diff
@@ -45,19 +43,18 @@ dnssec = []
 name = "trust_dns_resolver"
 path = "src/lib.rs"
 
-[build-dependencies]
-lalrpop = "^0.14.0"
-
 [dependencies]
+resolv-conf = "0.6.0"
 error-chain = "0.1.12"
 futures = "^0.1.17"
-lalrpop-util = "^0.14.0"
 lazy_static = "^1.0"
 log = "^0.4.1"
 lru-cache = "^0.1.1"
-regex = "0.2.1"
 tokio-core = "^0.1"
 trust-dns-proto = { version = "^0.2", path = "../proto" }
 
-[target.'cfg(windows)'.dependencies]
-ipconfig = { version = "^0.1.4" }
+[target.'cfg(all(windows, target_arch = "x86_64"))'.dependencies]
```

Comment:
Was this an intentional change? This is probably what’s cause the windows failure. Want to just revert this change?

Reply:
I rebased wrong code, I will revert this.